src/Bicep.LangServer.IntegrationTests/ParamsCompletionTests.cs

```diff
@@ -304,6 +304,64 @@ public async Task Using_completion_should_only_be_offered_once(string paramTextW
                     x.Kind.Should().Be(CompletionItemKind.Keyword);
                 });
         }
+        
+        [TestMethod]
+        public async Task Parameter_type_description_should_be_shown_for_params_symbol_completions()
+        {
+            var paramTextWithCursor = @"
+using './main.bicep'
+
+param |";
+
+            var bicepText = @"
+@allowed(
+    [
+        0
+        1
+    ]
+)
+@description('this is an int value')
+param myInt int
+
+@allowed(
+    [
+        'value1'
+        'value2'
+    ]
+)
+@description('this is a string value')
+param myStr string
+
+@description('this is a bool value')
+param myBool bool
+";
+            var fileTextsByUri = new Dictionary<Uri, string>
+            {
+                [InMemoryFileResolver.GetFileUri("/path/to/main.bicep")] = bicepText,
+            }; 
+        
+            var completions = await RunCompletionScenario(paramTextWithCursor , fileTextsByUri.ToImmutableDictionary(), '|');
+
+            completions.Should().SatisfyRespectively(
+                x =>
+                {
+                    x.Label.Should().Be("myBool");
+                    x.Documentation!.MarkupContent!.Value.Should().Be("Type: bool  \nthis is a bool value");
+                    x.Kind.Should().Be(CompletionItemKind.Field);
+                },
+                x =>
+                {
+                    x.Label.Should().Be("myInt");
+                    x.Documentation!.MarkupContent!.Value.Should().Be("Type: 0 | 1  \nthis is an int value");
+                    x.Kind.Should().Be(CompletionItemKind.Field);
+                },
+                x =>
+                {
+                    x.Label.Should().Be("myStr");
+                    x.Documentation!.MarkupContent!.Value.Should().Be("Type: 'value1' | 'value2'  \nthis is a string value");
```

Comment:
Could you also add an assertion for a param without a `@description()` decorator, just to make sure the documentation is correctly formatted?